tasks/team-access.yaml

```diff
@@ -3,85 +3,93 @@
 
 - include_vars: vars/acs-team-access-config.yaml
```

Comment:
`vars/acs-team-access-config.yaml` is ignored, update `.gitignore` to get it back. you don't need the sample one since it's just be part of the code base.

Reply:
updated.